monitor/docker.go

```diff
@@ -334,23 +324,15 @@ func (d *dockerMonitor) syncContainers() error {
 
 		if err != nil {
 			log.WithFields(log.Fields{
-				"package":   "monitor",
-				"monitor":   d,
-				"error":     err,
-				"container": container,
+				"package": "monitor",
+				"error":   err,
 			}).Error("Error Syncing existing Container")
-
```

Comment:
When syncing, if an error happens (Example the PolicyResolver fails), we simply do not activate the container in Trireme. Depending on the default policy, this means that the container will/will not be the open Network Connections